node_modules/oae-admin/tenants/tenants.html

```diff
@@ -19,6 +19,7 @@
                 {else}
                     <th>__MSG__HOST__</th>
                 {/if}
+                <th>__MSG__EMAIL_DOMAIN__</th>
```

Comment:
Lets keep it here. I will follow up with @SamPeck to come up with a better design for the tenant list that hides some of the metadata